astropy/stats/tests/test_funcs.py

```diff
@@ -256,3 +258,94 @@ def test_gaussian_sigma_to_fwhm():
 def test_gaussian_sigma_to_fwhm_to_sigma():
     assert_allclose(funcs.gaussian_fwhm_to_sigma *
                     funcs.gaussian_sigma_to_fwhm, 1.0)
+
```

Comment:
No luck so far chasing down concrete values of the Kuiper FPP. But the tests we do have directly test the desired property of the FPP calculation, that it reflects the known false positive probability. I can tighten them somewhat at the cost of increasing runtime.